### PR TITLE
Multi order book

### PR DESCRIPTION
**Pull request by juanszymczak**

### CODE REVIEW

**Comment by juanszymczak** [comment]
@tmsrjs ready!

File: spec/models/buy_opening_flow_spec.rb [hunk 1]
```diff
@@ -185,9 +185,9 @@
       end.to change { BitexBot::OpenBuy.count }.by(1)
     end
 
-    it 'does not register litecoin buys' do
+    it 'does not register buys from another order book' do
       flow.order_id.should == 12345
-      Bitex::Trade.stub(all: [build(:bitex_buy, id: 23456, specie: :ltc)])
+      Bitex::Trade.stub(all: [build(:bitex_sell, id: 23456, order_book: :btc_ars)])
```
Comment (tmsrjs) [inline spec/models/buy_opening_flow_spec.rb]: This stub should be a `Bitex::Buy` since the spec is for a `BuyOpeningFlow`

**Comment by juanszymczak** [comment]
[http://i0.kym-cdn.com/entries/icons/original/000/020/734/Lebron-James-Crying.jpg](url)

**Comment by juanszymczak** [comment]
crying

**Pull request merged**